networking/bgp-routing/about-bgp-routing.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-bgp-routing"]
+= About BGP routing
+include::_attributes/common-attributes.adoc[]
+:context: about-bgp-routing
+
+toc::[]
+
+This feature provides native BGP routing capabilities for the cluster. Additionally, this feature must be enabled to use the RouteAdvertisements feature of the OVN-Kubernetes network plugin.
```

Comment:
Coming in from a related diagram request - spell out acronym on first use? This isn't necessarily a ubiquitous term.